Legiones Astartes.cat

```diff
@@ -11129,7 +11131,7 @@ If at any point there are only Rapier Carrier Models remaining in this Unit, tho
         </selectionEntry>
       </selectionEntries>
       <categoryLinks>
-        <categoryLink targetId="cf96-8891-3f9a-8921" id="0f65-5063-27e0-d544" primary="false" name="Fast Attack"/>
```

Comment:
Not your fault, but this category shouldn't be here at all, since it's on the  link. Doesn't actually make a difference though (it's not counted twice).